MANIFEST.in

```diff
@@ -3,4 +3,5 @@ include versioneer.py
 include pygeos/_version.py
 include pygeos/*.dll
 include LICENSE
-exclude MANIFEST.in
\ No newline at end of file
+exclude MANIFEST.in
+exclude pyproject.toml
```

Comment:
I thought it should be included, not excluded? As I understand, it is necessary for `pip` to build an `sdist`.

Reply:
Are we expecting someone to create an `sdist` from something other than the full repo?  (in which case, it is included).

I may not fully understand the mechanics properly, but when we include it in `MANIFEST.in`, the distributed package installs Cython during its installation process, which isn't needed because we are shipping the C files produced by Cython.  When we exclude it, those C files are compiled during installation per our recent changes in `setup.py`, which worked properly when I tested installing from the source package (created using `sdist`) locally.